package.json

```diff
@@ -161,11 +170,11 @@
 		"lint-php": "docker-compose run --rm composer run-script lint",
 		"lint-pkg-json": "wp-scripts lint-pkg-json ./packages",
 		"package-plugin": "./bin/build-plugin-zip.sh",
-		"postinstall": "lerna bootstrap --hoist && npm run build:packages",
+		"postinstall": "lerna bootstrap --hoist && npm-run-all create-symlinks build:packages",
```

Comment:
I think this needs to use `run-s` rather than `npm-run-all` to run _ sequential_ rather than _ parallel_ which the is the default cc @gziolo 

See https://www.npmjs.com/package/npm-run-all#cli-commands

Reply:
It's sequential: https://github.com/mysticatea/npm-run-all/blob/HEAD/docs/npm-run-all.md#run-scripts-sequentially.
However, we should get rid of `npm-run-all`. We discussed it recently and agreed, that we already use another package for `parallel` execution and it's fine as is.

Reply:
Thanks, removed in https://github.com/WordPress/gutenberg/pull/7556/commits/a268477c4d35f45b520e0e5c325989a7eff8b1b9